### ISSUE TITLE
error: Unterminated string literal

### ISSUE THREAD

**Issue by ashsearle** [oven-sh/bun#52]
Bun 0.44 shows an error when parsing a long array of strings:
```
Error: SyntaxError

warn: Please run `bun bun` from a directory containing a package.json.

error: Unterminated string literal
```

Found with a small project:
```bash
yarn add faker
```

Create `index.js`:
```js
require('faker');
```

This then errors:
```bash
bun bun ./index.js
```

**Comment by Jarred-Sumner** [comment]
Thanks. I'm looking into this now.

> 'Esa�',

It looks like it's an issue with invalid unicode characters, however the line number is incorrect

**Comment by Jarred-Sumner** [comment]
Fixed in fc59a32b636404f7d9e8706c44c92761a1873318
